model/group_request.go

```diff
@@ -25,18 +25,22 @@ func NewCreateGroupRequestFromReader(reader io.Reader) (*CreateGroupRequest, err
 		return nil, errors.Wrap(err, "failed to decode create group request")
 	}
 
-	if createGroupRequest.Name == "" {
-		return nil, errors.New("must specify name")
-	}
-	if createGroupRequest.Version == "" {
-		return nil, errors.New("must specify version")
+	err = createGroupRequest.Validate()
```

Comment:
:+1: I feel like you told me to move this line here during my PR so my apologies for missing it, assuming that's what happened. Regardless, I'm happy to see this moved.

Reply:
No worries! It's there now.